posthog/schema.py

```diff
@@ -8826,6 +8826,15 @@ class DatabaseSchemaManagedViewTable(BaseModel):
     source_id: Optional[str] = None
     type: Literal["managed_view"] = "managed_view"
 
+    def description(self) -> Optional[str]:
+        if self.kind == DatabaseSchemaManagedViewTableKind.REVENUE_ANALYTICS_CHARGE:
+            return "Useful for Revenue-related questions, contains charges for customers"
```

Comment:
"charges for customers" -> "charges per customer"?

Reply:
Yeah, not really. It's all the charges we've ever received, and they *might* map to a customer. Will work on improving this